CMakeLists.txt

```diff
@@ -17,42 +17,109 @@
 ###############################################################################
 
 cmake_minimum_required(VERSION 3.15)
-cmake_policy(SET CMP0091 NEW)
-
-set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded")
-set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /NODEFAULTLIB:MSVCRT /NODEFAULTLIB:MSVCRTD /NODEFAULTLIB:LIBCMTD /DEBUG:FULL /FORCE:MULTIPLE /IGNORE:4006,4075,4088,4099")
-
-set(CMAKE_MODULE_PATH "${CMAKE_BINARY_DIR}/vendor;${CMAKE_SOURCE_DIR}/cmake")
-
-option(SUPERBUILD "Build the project using a superbuild" ON)
-
-if (SUPERBUILD)
-	project(Makou_Reactor_SUPERBUILD LANGUAGES CXX)
-	set_directory_properties(PROPERTIES EP_BASE "${CMAKE_BINARY_DIR}/ep")
-	add_subdirectory(third_party)
-	include(ExternalProject)
-	ExternalProject_Add(
-		Makou_Reactor
-		SOURCE_DIR	"${CMAKE_SOURCE_DIR}"
-		CMAKE_ARGS
-			"-DSUPERBUILD=OFF"
-			"-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
-			"-DCMAKE_PREFIX_PATH=${CMAKE_BINARY_DIR}/vendor"
-			"-DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}"
-			"-DCMAKE_MSVC_RUNTIME_LIBRARY=${CMAKE_MSVC_RUNTIME_LIBRARY}"
-		DEPENDS
-		zlib
-	)
-	ExternalProject_Add_Step(
-		Makou_Reactor
-		reconfigure
-		COMMAND ${CMAKE_COMMAND} -E echo "Forcing a superbuild reconfigure"
-		DEPENDEES download
-		DEPENDERS configure
-		ALWAYS ON
-	)
-	return()
+cmake_policy(SET CMP0074 NEW)
+
+set(RELEASE_NAME "Makou_Reactor")
+set(GUI_TARGET "${RELEASE_NAME}")
+#set(CLI_TARGET "makoureactor")
+
+project(${RELEASE_NAME} VERSION 1.8.5 LANGUAGES CXX)
+
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_compile_definitions(QT_DISABLE_DEPRECATED_BEFORE=0x050E00)
+
+find_package(QT NAMES Qt5 Qt6 COMPONENTS Widgets OpenGL Zlib REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGL Zlib REQUIRED)
+find_package(ff7tk COMPONENTS ff7tk ff7tkUtils REQUIRED)
+
+file(GLOB_RECURSE PROJECT_SOURCES "src/*.cpp" "src/*.h")
+set(RESOURCES "src/qt/${RELEASE_NAME}.qrc")
+
+if (APPLE)
+    set(OS_BUNDLE MACOSX_BUNDLE)
```

Comment:
Both WIN32 and MACOSX_BUNDLE can be  in the same add_executable. They only have an effect on their platforms and no others.